server/views.py

```diff
@@ -35,16 +35,26 @@ def get_mandatory_areas(request):
 
 @login_required
 def index(request):
+    now = datetime.datetime.now()
+
+    last_7am = datetime.datetime(now.year, now.month, now.day - (1 if now.hour < 7 else 0), 7, 0, 0)
+    next_7am = last_7am + datetime.timedelta(days=1)
     context = {
         "numbers": {
             "total_volunteers": Volunteer.objects.count(),
             "total_help_requests": HelpRequest.objects.count(),
             "solved_help_requests": HelpRequest.objects.filter(status="DONE").count()
         },
         "daily_numbers": {
-            "daily_volunteers": 12,
-            "daily_help": 12,
-            "daily_solved": 12
+            "daily_volunteers": Volunteer.objects.filter(
+                Q(created_date__range=(last_7am,
+                                       next_7am))).count(),  # Tomorrow
+            "daily_help": HelpRequest.objects.filter(
+                Q(created_date__range=(last_7am,
+                                       next_7am))).count(),  # Tomorrow
+            "daily_solved": HelpRequest.objects.filter(Q(created_date__range=(last_7am,
+                                                                              next_7am)),
+                                                       status="DONE").count()
```

Comment:
The readability here is very limited,
can you explain what are you doing here?

regardless think of a way to make this code readable

Reply:
Improved, also added comment.